doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/configs/xgboost-benchmark.yaml

```diff
@@ -0,0 +1,48 @@
+# This is a Ray cluster configuration for exploration of the 100Gi XGBoost-Ray workload.
```

Comment:
I'm fine with that. I did this because it's the structured used by k8s stack. cc @DmitriGekhtman

Reply:
Actually it might be better to put all configs in `configs` folder similar to all code is in `doc_code` folder. This way, later we can test all configs in this folder through CI.

Reply:
True. I'm fine deferring it to later.